app/controllers/admin_controller.rb

```diff
@@ -40,6 +40,14 @@ def transaction
     @canonical_pending_transactions = CanonicalPendingTransaction.unmapped.where(amount_cents: @canonical_transaction.amount_cents)
     @ahoy_events = Ahoy::Event.where("name in (?) and (properties->'canonical_transaction'->>'id')::int = ?", [::SystemEventService::Write::SettledTransactionMapped::NAME, ::SystemEventService::Write::SettledTransactionCreated::NAME], @canonical_transaction.id).order("time desc")
 
+    if @canonical_transaction.memo.include?("WISE INC")
+      potential_wise_transfers = WiseTransfer.sent.where(amount_cents: @canonical_transaction.amount_cents)
```

Comment:
Is this `amount_cents` in USD?

Reply:
Oops, I fixed this locally but forgot to push